docs/DeveloperGuide.md

```diff
@@ -433,7 +440,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * **Mainstream OS**: Windows, MacOS, Unix
 * **Meeting**: An event that the user with the client at a specific date and time.
-
+* **Timing conflict**: Time periods that overlap. e.g. meetings should not have any timing conflict.
```

Comment:
you are using timing conflict to explain timing conflict which is a bit confusing to me, maybe a better approach will be `meetings should not be happening on the same time`?

Reply:
I can see where the confusion may appear, have changed